.github/workflows/hive.yaml

```diff
@@ -0,0 +1,239 @@
+name: Hive - Fusaka
+on:
+  schedule:
+    - cron: '45 12 * * *'
+  workflow_dispatch:
+    # Note: We're limited to 10 inputs
+    inputs:
+      client:
+        type: string
+        default: '"go-ethereum","reth","nethermind","ethereumjs","besu","erigon"'
+        description: Comma-separated list of clients to test .e.g go-ethereum, besu, reth, nethermind, erigon, ethereumjs
+      simulator:
+        type: string
+        default: '"ethereum/eest/consume-engine","ethereum/eest/consume-rlp"'
+        description: Comma-separated list of simulators to test ethereum/eest/consume-engine, ethereum/eest/consume-rlp
+      hive_version:
+        type: string
+        default: ethereum/hive@master
+        description: GitHub repository and tag for hive (repo@tag)
+      geth_version:
+        type: string
+        default: ethereum/go-ethereum@eof-devnet-0
+        description: GitHub repository and tag for go-ethereum (repo@tag)
+      besu_version:
+        type: string
+        default: hyperledger/besu@main
+        description: GitHub repository and tag for besu (repo@tag)
+      reth_version:
+        type: string
+        default: paradigmxyz/reth@main
+        description: GitHub repository and tag for reth (repo@tag)
+      nethermind_version:
+        type: string
+        default: NethermindEth/nethermind@feature-evm-eof
+        description: GitHub repository and tag for nethermind (repo@tag)
+      erigon_version:
+        type: string
+        default: erigontech/erigon@eof-tests
+        description: GitHub repository and tag for erigon (repo@tag)
+      ethereumjs_version:
+        type: string
+        default: ethereumjs/ethereumjs-monorepo@master
+        description: GitHub repository and tag for ethereumjs (repo@tag)
+
+env:
+  # Proxy
+  GOPROXY: "${{ vars.GOPROXY }}"
+  # Hive action environment variables
+  S3_BUCKET: hive-results
+  S3_PATH: fusaka
```

Comment:
I'd go with `eof-devnet-0` to keep it clear which spec is being tested, for now.